packages/mephisto-task/README.md

```diff
@@ -102,6 +102,26 @@ Generally speaking, the value is the `InitializationData` object that the `TaskR
 
 A callback provided for the webapp to finalize and submit the worker's resulting work back to Mephisto.
 
+### `handleMetadataSubmit(payload)`
+A callback provided for the Tips component in the `mephisto-worker-experience` package to submit a tip to its agent's metadata. 
+Not recommended to be used by the user as mephisto-worker-experience should use this for you under the hood.
```

Comment:
The `handleMetadataSubmit` callback is generic and can be used for any future addon, not just Tips, Feedback or `mephisto-worker-addons` library. Though you developed it in conjunction with these, we should keep the documentation here vague to indicate to users at a more generic level what this method does: persists agent-level metadata in the AgentState.